cclib/parser/molcasparser.py

```diff
@@ -696,6 +696,30 @@ def extract(self, inputfile, line):
                     self.mocoeffs = []
                 self.mocoeffs.append(mocoeffs)
 
+        ## Parsing MP energy from the &MBPT2 module.
+        #  Conventional algorithm used...
+        #
+        #         SCF energy                           =      -74.9644564043 a.u.
+        #         Second-order correlation energy      =       -0.0364237923 a.u.
+        #
+        #         Total energy                         =      -75.0008801966 a.u.
+        #         Reference weight ( Cref**2 )         =        0.98652
+        #
+        #  ::    Total MBPT2 energy                              -75.0008801966
+        #
+        #
+        #         Zeroth-order energy (E0)             =      -36.8202538520 a.u.
+        #
+        #         Shanks-type energy S1(E)             =      -75.0009150108 a.u.
+        if 'Total MBPT2 energy' in line:
+            mpenergies = []
+            mpenergies.append(utils.convertor(self.float(line.split()[4]), 'hartree', 'eV'))
+            self.skip_lines(inputfile, ['b', 'b', 'zeroth order energy', 'b'])
```

Comment:
Do you need to skip, and parse the additional line after this, after parsing what you need?

Reply:
My bad. I thought that we needed to parse the Shanks type enregy as well so I wrote this. Will remove it.